approval_polls/tests.py

```diff
@@ -8,14 +8,16 @@
 
 from approval_polls.models import Poll
 
-def create_poll(question, days=0, ballots=0):
+def create_poll(question, username, days=0, ballots=0):
```

Comment:
I've been tried this and it cause username not unique if create_poll has been called twice as follows: 
create_poll(question="Past poll.", username="user1", days=-30)
create_poll(question="Future poll.", username="user2", days=30)

Let me know if you have different thought.
Thanks!